docs/tutorials/creating-your-first-substrate-chain/test.md

```diff
@@ -0,0 +1,187 @@
+      });
+
+    return () => unsubscribe && unsubscribe();
+  }, [digest, api.query.templateModule]);
+
+  function isClaimed() {
+    return block !== 0;
+  }
+
+  return (
+    <Grid.Column>
+      <h1>Proof Of Existence</h1>
+      <Form success={!!digest && !isClaimed()} warning={isClaimed()}>
+        <Form.Field>
+          <Input
+            type="file"
+            id="file"
+            label="Your File"
+            onChange={e => handleFileChosen(e.target.files[0])}
+          />
+          <Message success header="File Digest Unclaimed" content={digest} />
+          <Message
+            warning
+            header="File Digest Claimed"
+            list={[digest, `Owner: ${owner}`, `Block: ${block}`]}
+          />
+        </Form.Field>
+
+        <Form.Field>
+          <TxButton
+            accountPair={accountPair}
+            label={"Create Claim"}
+            setStatus={setStatus}
+            type="TRANSACTION"
+            attrs={{ params: [digest], tx: api.tx.templateModule.createClaim }}
+            disabled={isClaimed() || !digest}
+          />
+          <TxButton
+            accountPair={accountPair}
+            label="Revoke Claim"
+            setStatus={setStatus}
+            type="TRANSACTION"
+            attrs={{ params: [digest], tx: api.tx.templateModule.revokeClaim }}
+            disabled={!isClaimed() || owner !== accountPair.address}
+          />
+        </Form.Field>
+        <div style={{ overflowWrap: "break-word" }}>{status}</div>
+      </Form>
+    </Grid.Column>
+  );
+}
+```
+
+We won't walk you step by step through the creation of this component, but do
+look over the code comments to learn what each part is doing.
+
+## Submit a Proof
+
+Your front-end should have automatically restarted with this new component
+included. Select any file on your computer, and you will see that you can create a claim
+with it's file digest:
+
+![Proof Of Existence Component](assets/poe-component.png)
+
+If you press "Create Claim", a transaction will be dispatched to your custom
+Proof of Existence module, where this digest and the selected user account will
+be stored on chain.
+
+![Claimed File](assets/poe-claimed.png)
+
+If all went well, you should see a new `ClaimCreated` event appear in the Events
+component. The front-end can automatically recognize that your file is now
+claimed, and even gives you the option to revoke the claim if you want.
+
+Remember, only the owner can revoke the claim! If you select another user
+account at the top, and you will see that the revoke option is disabled!
+
+## Next Steps
+
+This is the end of our journey into creating a Proof of Existence blockchain.
+
+You have seen first hand how simple it can be to develop a brand new runtime
+module and launch a custom blockchain using Substrate. Furthermore, we have
+shown you that the Substrate ecosystem provides you with the tools to quickly
+and easily create responsive front-end experiences so users can interact with
```

Comment:
```suggestion
create responsive front-end experiences so users can interact with
```

I'd refrain from calling anything in learning materials `easy`(or `just do XYZ` for that matter). I know this is a bit more of a marketing place, but I'd still do it. Just consider if it failed for someone, they weren't able to get it to work but go on and then read that it was easy to do ...

Reply:
I am 100% with you here, and love for you to call me out whenever I do it.

I write like a talk, which is sometimes more passionate than is helpful.